src/sdk/pynni/nni/hyperband_advisor/hyperband_advisor.py

```diff
@@ -296,6 +296,11 @@ def __init__(self, R, eta=3, optimize_mode='maximize'):
         # In this case, tuner increases self.credit to issue a trial config sometime later.
         self.credit = 0
 
+        # record the latest parameter_id of the trial job trial_job_id.
+        # if there is no running parameter_id, self.job_id_para_id_map[trial_job_id] == None
+        # new trial job is added to this dict and finished trial job is removed from it.
+        self.job_id_para_id_map = dict()
+
     def load_checkpoint(self):
         pass
```

Comment:
load_checkpoint and save_checkpoint  can be removed if they are empty.

Reply:
removed, thx!